story/story_manager.py

```diff
@@ -206,8 +206,8 @@ def load_from_storage(self, story_id):
         else:
             return None
 
-    def save_story(self):
-        story_id = str(uuid.uuid1())
+    def save_story(self, name=None):
+        story_id = str(uuid.uuid1()) if name is None else ("_" + name)

```

Comment:
Why add the underscore? When saving, the filename will end up as "story_FileName.json", but load_from_storage will be looking for `file_name = "story" + story_id + ".json"`, i.e. "storyFileName.json" (code for handling encryption left out for brevity).

Reply:
yeah... backwards compatibility is a must, so either we load with an underscore "_storyname" or the filenames look like "storystoryname.json"

Reply:
But you're right, who cares what the filename is, I got rid of that so that you load the same string you save with.